src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/PatientClinicalInfo.java

```diff
@@ -0,0 +1,46 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup;
+
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.ClinicalInformationFreeText;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * Provides clinical information about the patient.
+ * <p>
+ * Segment group 10: {@code S10-CIN-DTM-FTX-SG14}
+ * <p>
+ * Parents: {@link uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message Message}
+ * &gt; {@link ServiceReportDetails}
+ * &gt; {@link InvestigationSubject}
+ */
+public class PatientClinicalInfo extends SegmentGroup {
+    public static final String INDICATOR = "S10";
+
+    // CIN - all contents marks as "N"ot used
```

Comment:
Ah, I guess I misunderstood the document. It itself must be given, but almost all of its _fields_ are not used for us. All I have in my examples is `CIN+UN'`.

I could add it, but perhaps we can understand as a business rule that its value is always `UN` (unspecified) and if it has any bearing on the FHIR output, it will just be hard-coded?

Reply:
Do we have other examples of segments that are always the same but we've made classes for them anyway?

Reply:
Ah yeah, i see what you mean there - all of the fields for `CIN`, other than `code`, are not used. Maybe adding this to the javadoc would be good enough then?

Reply:
I can't think of any examples right now

Reply:
The only example i can think of is the `DiagnosticReportCode.java` / `GIS` segment where the only field, `code` is always set to "N"

Reply:
Hmm yeah. That's enough of a precedent - will add the class

Reply:
Now added